src/Components/InspoContainer/InspoContainer.scss

```diff
@@ -116,3 +116,33 @@ h1 {
 .hidden {
   display: none;
 }
+
+@media screen and (min-device-width: 0px) and (max-device-width: 1205px) {
+  .question-button,
+  .questionMark-text {
+    height: 30px;
+    width: 30px;
+    position: relative;
+  }
+  .pencil-button,
+  .pencil-img {
```

Comment:
The text no longer appears, because there is no hovering on a touch screen. Do we want to make the text always visible at smaller screen sizes?